internal/pkg/helm/values_inspector.go

```diff
@@ -91,3 +100,141 @@ func isBooleanYamlValue(input *map[string]interface{}, path *[]string, pathIndex
 		}
 	}
 }
+
+//GetValues .. Get values  fro given path
+func GetValues(arg string) (*map[string][]*LogHelmReport, error) {
+	argString := strings.ReplaceAll(arg, ".Values.", "")
+	chartClient := NewChartClient()
+	err := chartClient.LoadChartFrom(HelmChartRef)
+	if err != nil {
+		logrus.Warnf("error loading chart: %s", err)
+		return nil, err
+	}
+	raw, _ := chartutil.ReadValues([]byte(chartClient.Chart.Values.Raw))
+	result, err := raw.PathValue(argString)
+	if err != nil {
+		logrus.Warnf("Path value not found for path : %s ,", argString)
+		return nil, err
+	}
+	return dump(result), nil
+}
+
+//Converts and dumps the data of type interface{} slice
+//returned by chartUtil.GetPathValues into *map[string]interface{}
+func dump(result interface{}) *map[string][]*LogHelmReport {
+	items := reflect.ValueOf(result)
+	returnMap := make(map[string][]*LogHelmReport)
+	if items.Kind() == reflect.Slice {
+		for i := 0; i < items.Len(); i++ {
+			item := items.Index(i)
+			if item.Elem().Kind() == reflect.String {
+				returnMap[item.Interface().(string)] = []*LogHelmReport{}
+			} else if item.Elem().Kind() == reflect.Map {
+				returnMaps := item.Interface().(map[string]interface{})
+				for k := range returnMaps {
+					returnMap[k] = []*LogHelmReport{}
+				}
+			}
+		}
+	}
+	return &returnMap
+}
+
+//PreFixValuesWithItems...For single item list {% for item in hosts %}{{ item.name }}{% endfor %},
+//This function will prefix the list variables with a loop variable, within the body of the for loop.
+//There is two cases , one with field name that needs to be prefixed with loop variables
+// other condition is dot field name is replaced by loop variable
```

Comment:
Inconsistent spacing;  Please ensure all lines for API comments for exported methods start with a space.  I.e.,
```
// PreFixValuesWithItems...For single item list {% for item in hosts %}{{ item.name }}{% endfor %},
// This function will prefix the list variables with a loop variable, within the body of the for loop.
// There is two cases , one with field name that needs to be prefixed with loop variables
// other condition is dot field name is replaced by loop variable
```

Reply:
We need to enable this rule in golint. I feel the fix should  be in the linter